.github/workflows/linux_ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       run: cargo build --verbose
 
     - name: Run tests
-      run: cargo test --verbose
+      run: cargo test --all --verbose
```

Comment:
Isn't the "--all" parameter deprecated? 

according to https://doc.rust-lang.org/cargo/commands/cargo-test.html : 

> `--workspace`
> Test all members in the workspace.
> `--all`
> Deprecated alias for `--workspace`.

Reply:
Good shot! Fixed.